scripts/install-pw.js

```diff
@@ -24,10 +24,10 @@ THE SOFTWARE.
 
 // Load the process.env config from .env file
 require('dotenv').config();
-
 const { Registry } = require('playwright/lib/utils/registry');
-const SYNTHETICS_BROWSER_REVISIONS = require('@elastic/synthetics/node_modules/playwright-chromium/browsers.json');
-
+const PLAYWRIGHT_CORE_PATH = require.resolve('playwright-core');
+const BROWSER_PATH = PLAYWRIGHT_CORE_PATH.replace('index.js', 'browsers.json');
```

Comment:
The reason why we want to extract the browser version from Synthetics agent is because we want the recorder to record and run results in the same exact browser version. 

PW browser.json depends on why we updated the fork and might not match the Synthetics test version we want to use.

Reply:
So it needs to be `@elastic/synthetics/node_modules/playwright-core/browsers.json`

Reply:
What I've seen with the newer npm is that it downloads `playwright-core` to top-level `node_modules` rather than `@elastic/synthetics/node_modules` (`@elastic/synthetics/node_modules` doesn't have `playwright-core`):

<img width="307" alt="image" src="https://user-images.githubusercontent.com/16660866/183613220-de55db94-985a-4b4a-8ff1-c42ee9c77419.png">
The forked PW is downloaded and aliased as `playwright` so I think we still are importing the `browser.json` from the `playwright-core` package that is used by Synthetics agent.

Reply:
Ahh, npm 7 has workspaces set by default. Good callout, can you add a comment for the same - Its hard to grok in.

Reply:
Sure, added!!